README.it.md

```diff
@@ -0,0 +1,183 @@
+<br>
+
+<div align="center">
+
+[<img src="./resources/icon.png" width="144"/>](https://hydralauncher.site)
+
+  <h1 align="center">Hydra Launcher</h1>
+  
+  <p align="center">
+    <strong>Hydra è un game launcher con il proprio client bittorrent e autogestore di repacks.</strong>
+  </p>
+
+[![build](https://img.shields.io/github/actions/workflow/status/hydralauncher/hydra/build.yml)](https://github.com/hydralauncher/hydra/actions)
+[![release](https://img.shields.io/github/package-json/v/hydralauncher/hydra)](https://github.com/hydralauncher/hydra/releases)
+
+[![be](https://img.shields.io/badge/lang-be-orange)](README.be.md)
+[![pl](https://img.shields.io/badge/lang-pl-white)](README.pl.md)
+[![pt-BR](https://img.shields.io/badge/lang-pt--BR-green.svg)](README.pt-BR.md)
+[![ru](https://img.shields.io/badge/lang-ru-yellow.svg)](README.ru.md)
+[![uk-UA](https://img.shields.io/badge/lang-uk--UA-blue)](README.uk-UA.md)
+[![es](https://img.shields.io/badge/lang-es-red)](README.es.md)
+[![ita](https://img.shields.io/badge/lang-it-red)](README.it.md)
```

Comment:
must add the link for this readme on other readmes also

Reply:
if possible, also keep them in the same order on all files, so that can close issue #471